src/netlink.h

```diff
@@ -0,0 +1,24 @@
+#include "timeout.h"
+#include "lua.h"
+#include "socket.h"
+
+#define MAX_PAYLOAD 2048
+
+
+/*=========================================================================*\
+* .h definitions here because i'm lazy lmao 
+\*=========================================================================*/
+
+#define t_pid int
+#define t_groups int
+#define t_type int
+
+typedef struct t_netlink_{
```

Comment:
Do you use `struct t_netlink_` somewhere? If not, the struct doesn't need a name.